test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,68 @@
+package seedu.addressbook.commands;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.*;
```

Comment:
Don't import everything using *; Import only the (necessary) classes explicitly.